modal/_utils/async_utils.py

```diff
@@ -771,7 +833,13 @@ async def worker() -> AsyncGenerator[V, None]:
         while True:
             item = await queue.get()
             if isinstance(item, ValueWrapper):
-                yield await async_mapper_func(item.value)
+                if isinstance(item.value, int) and item.value % 500 == 0:
+                    print(f"{item.value} found at time {time.time() - t0}")
+                r = await async_mapper_func(item.value)
+                if yield_nothing:
```

Comment:
Would this actually make any difference? If the mapper function doesn't have a result it would return None, and if it did have a result it would be passed by reference and wouldn't be slower than passing None afaik?